docs/reference/vacuumgripper.md

```diff
@@ -0,0 +1,254 @@
+## Vacuum Gripper
+
+Derived from [Device](device.md) and [Solid](solid.md).
+
+```
+VacuumGripper {
+  SFBool   isOn              FALSE         # {TRUE, FALSE}
+  SFFloat  tensileStrength   -1            # {-1, [0, inf)}
+  SFFloat  shearStrength     -1            # {-1, [0, inf)}
+  SFInt32  contactPoints     3             # [1, inf)
+}
+```
+
+### Description
+
+[VacuumGripper](#vacuum-gripper) node is used to simulate physical links created by vacuum suction.
+
+[VacuumGripper](#vacuum-gripper) node can only connect to dynamic [Solid](solid.md) nodes.
```

Comment:
This sentence should be clarified. Does it mean it can connect only to Solid nodes with a Physics node? If so, that sounds wrong to me as it could be that a legged robot with vacuum grippers in its feet connect to the ground which has no physics (static environment).

Reply:
Yes, currently it works only for Solid nodes with a Physics node (as it is also the case for the Connector node).
I will check to change it.